core/src/main/resources/defaults/blueprints/7.2.10/cdp-data-engineering-ha.bp

```diff
@@ -147,7 +147,8 @@
             "name": "hive_service_config_safety_valve",
             "value": "<property><name>fs.s3a.ssl.channel.mode</name><value>openssl</value></property><property><name>hive.txn.acid.dir.cache.duration</name><value>0</value></property><property><name>hive.server2.tez.session.lifetime</name><value>30m</value></property>
                         <property><name>hive.blobstore.supported.schemes</name><value>s3,s3a,s3n,abfs,gs</value></property><property><name>hive.orc.splits.include.fileid</name><value>false</value></property><property><name>hive.hook.proto.events.clean.freq</name><value>1h</value></property>
-                        <property><name>hive.metastore.try.direct.sql.ddl</name><value>true</value></property>"
+                        <property><name>hive.metastore.try.direct.sql.ddl</name><value>true</value></property>
```

Comment:
Looks like this value was not propagated to HMS as shown by @amitagarwal06 .

You could set this value in HMS safety vavle and try to export the template using CM API. A nice tool is available in CM UI left hand side support tab.